src/Signup/components/Signup.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Image, View, Text, TouchableOpacity, TextInput } from 'react-native';
+import { PropTypes } from 'prop-types';
+
+import styles from '../components/Signup.style';
+
+const logoImg = require('../../globals/EventAllLightLong.png');
+
+class SignupScreen extends React.Component {
+  static propTypes = {
+    navigation: PropTypes.shape({
+      navigate: PropTypes.func.isRequired,
+    }).isRequired,
+  };
+
+  constructor() {
```

Comment:
Same thing as above: we can initialize state by just directly setting a state object